src/cmd/flux-jobs.py

```diff
@@ -32,15 +32,15 @@ class FluxJobsConfig(UtilConfig):
         "default": {
             "description": "Default flux-jobs format string",
             "format": (
-                "{id.f58:>12} ?:{queue:<8.8} {username:<8.8} {name:<10.10} "
+                "{id.f58:>12} ?:{queue:<8.8} {username:<8.8} {name:<10.10+} "
                 "{status_abbrev:>2.2} {ntasks:>6} {nnodes:>6h} "
                 "{contextual_time!F:>8h} {contextual_info}"
             ),
         },
         "long": {
             "description": "Extended flux-jobs format string",
             "format": (
-                "{id.f58:>12} ?:{queue:<8.8} {username:<8.8} {name:<10.10} "
+                "{id.f58:>12} ?:{queue:<8.8} {username:<8.8} {name:<10.10+} "
```

Comment:
idea: wider field for name under long format to lower odds of truncation?

Reply:
yeah, i viewed it as a "maybe stick into this PR?"  I was not super won over by the idea, just threw it out.  We can just keep in the back of our minds.